pulp_smash/utils.py

```diff
@@ -124,6 +125,22 @@ def get_importers(server_config, href, responses=None):
     ), responses)
 
 
+def get_repo(server_config, href, responses=None):
```

Comment:
FYI, this method can be used to get any sort of resource from the server. Can we rename it and update its docstring to reflect its generic nature?